### DISCUSSION TITLE
Allow assigning issues to teams (like reviewers to pull requests)

### DISCUSSION THREAD

**Discussion by comprost** [community/community#127295] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

This feature request: https://github.com/cli/cli/issues/3932

Summary:
Request to enhance GitHub Actions to support assigning GitHub Teams to issues, alongside individual users, for improved collaboration and streamlined workflow management.

Description:
Currently, GitHub Actions only supports assigning individual users to issues. This limitation creates challenges for projects that rely heavily on team-based collaboration. Enabling the ability to assign GitHub Teams to issues would greatly enhance the flexibility and efficiency of managing issues within organizations.

Use Case:
Many organizations and projects operate using teams rather than relying on specific individuals for tasks. For example, a development team might need to address an issue collectively, ensuring that any available team member can take ownership and action. Assigning issues to teams would facilitate better workload distribution, reduce bottlenecks, and enhance overall productivity.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ccocks** [comment]
We need this!

**Comment by zsloane** [comment]
How is this still not possible. This is like a core tenant of many workflows
